SimCalorimetry/HGCalSimProducers/interface/HGCFEElectronics.h

```diff
@@ -43,13 +43,27 @@ class HGCFEElectronics
       }
   }
 
+
+  void SetNoiseValues(std::vector<float> noise_fC){
+    for( auto noise : noise_fC ) { noise_fC_.push_back( noise ); }
+  };
+
+  float getTimeJitter(float totalCharge, int thickness){
+    float A2 = jitterNoise2_ns_[thickness-1];
+    float C2 = jitterConstant2_ns_[thickness-1];
+    float X2 = pow((totalCharge/noise_fC_[thickness-1]), 2.);
+    float jitter2 = A2 / X2 + C2;
+    return sqrt(jitter2);
+  };
+
   /**
      @short returns the LSB in MIP currently configured
   */
   float getADClsb()       { return adcLSB_fC_;       }
   float getTDClsb()       { return tdcLSB_fC_;       }  
   float getADCThreshold() { return adcThreshold_fC_; }
   float getTDCOnset()     { return tdcOnset_fC_;     }
+  std::array<float,3> getTDCForToaOnset()    { return tdcForToaOnset_fC_; }
```

Comment:
In `HGCSample`, accessors are written with "ToA", but here (and in `HGCDigitizerBase`) accessors are written with "Toa". Please use consistent capitalization.